src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const ONE_DAY_RENT_COST = 40;
+  const THREE_DAYS_RENT_DISCOUNT = 20;
+  const SEVEN_DAYS_RENT_DISCOUNT = 50;
+  const SMALL_TERM_RENT = 3;
+  const LONG_TERM_RENT = 7;
+  const TOTAL_RENT = ONE_DAY_RENT_COST * days;
```

Comment:
This variable should be named in camelCase
`UPPER_CASE` -> constants(as above, lines 31-35)
`camelCase` -> calculated value